src/lib/cli.ts

```diff
@@ -103,6 +103,13 @@ yargs
     type: 'string',
     nargs: 1,
   })
+  .option('o', {
+    alias: 'overwrite',
+    default: true,
+    describe: 'Overwrite remote files with the same name, or skip them.',
+    type: 'boolean',
+    nargs: 1,
```

Comment:
`nargs` is not needed for booleans, only for strings that could require multiple values